2020-12-24 11:15:29.401 febs [http-nio-9527-exec-5] ERROR c.m.f.c.authentication.JWTFilter - token已经过期
2020-12-24 11:15:29.404 febs [http-nio-9527-exec-5] ERROR c.m.f.c.authentication.JWTFilter - token已经过期
2020-12-24 20:12:32.488 febs [main] ERROR c.m.f.common.runner.CacheInitRunner - 缓存初始化失败，Could not get a resource from the pool
2020-12-24 20:12:32.488 febs [main] ERROR c.m.f.common.runner.CacheInitRunner -  ____   __    _   _ 
2020-12-24 20:12:32.488 febs [main] ERROR c.m.f.common.runner.CacheInitRunner - | |_   / /\  | | | |
2020-12-24 20:12:32.488 febs [main] ERROR c.m.f.common.runner.CacheInitRunner - |_|   /_/--\ |_| |_|__
2020-12-24 20:12:32.491 febs [main] ERROR c.m.f.common.runner.CacheInitRunner -                         
2020-12-24 20:12:32.492 febs [main] ERROR c.m.f.common.runner.CacheInitRunner - FEBS启动失败              
2020-12-24 20:12:32.492 febs [main] ERROR c.m.f.common.runner.CacheInitRunner - Redis连接异常，请检查Redis连接配置并确保Redis服务已启动
2020-12-24 20:54:21.457 febs [http-nio-9527-exec-2] ERROR c.m.f.c.authentication.JWTFilter - token已经过期
2020-12-24 20:54:21.492 febs [http-nio-9527-exec-2] ERROR c.m.f.c.authentication.JWTFilter - token已经过期
2020-12-24 21:05:56.862 febs [QuartzScheduler_MyScheduler-DESKTOP-PGNP2U71608815127084_MisfireHandler] ERROR o.s.s.quartz.LocalDataSourceJobStore - MisfireHandler: Error handling misfires: Couldn't retrieve trigger: No record found for selection of Trigger with key: 'DEFAULT.TASK_11' and statement: SELECT * FROM QRTZ_CRON_TRIGGERS WHERE SCHED_NAME = 'MyScheduler' AND TRIGGER_NAME = ? AND TRIGGER_GROUP = ?
org.quartz.JobPersistenceException: Couldn't retrieve trigger: No record found for selection of Trigger with key: 'DEFAULT.TASK_11' and statement: SELECT * FROM QRTZ_CRON_TRIGGERS WHERE SCHED_NAME = 'MyScheduler' AND TRIGGER_NAME = ? AND TRIGGER_GROUP = ?
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.retrieveTrigger(JobStoreSupport.java:1538)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.recoverMisfiredJobs(JobStoreSupport.java:984)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.doRecoverMisfires(JobStoreSupport.java:3264)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$MisfireHandler.manage(JobStoreSupport.java:4012)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport$MisfireHandler.run(JobStoreSupport.java:4033)
Caused by: java.lang.IllegalStateException: No record found for selection of Trigger with key: 'DEFAULT.TASK_11' and statement: SELECT * FROM QRTZ_CRON_TRIGGERS WHERE SCHED_NAME = 'MyScheduler' AND TRIGGER_NAME = ? AND TRIGGER_GROUP = ?
	at org.quartz.impl.jdbcjobstore.CronTriggerPersistenceDelegate.loadExtendedTriggerProperties(CronTriggerPersistenceDelegate.java:107)
	at org.quartz.impl.jdbcjobstore.StdJDBCDelegate.selectTrigger(StdJDBCDelegate.java:1819)
	at org.quartz.impl.jdbcjobstore.JobStoreSupport.retrieveTrigger(JobStoreSupport.java:1536)
	... 4 common frames omitted
2020-12-24 21:06:36.261 febs [main] ERROR o.apache.catalina.util.LifecycleBase - Failed to start component [Connector[HTTP/1.1-9527]]
org.apache.catalina.LifecycleException: Protocol handler start failed
	at org.apache.catalina.connector.Connector.startInternal(Connector.java:1008)
	at org.apache.catalina.util.LifecycleBase.start(LifecycleBase.java:183)
	at org.apache.catalina.core.StandardService.addConnector(StandardService.java:227)
	at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.addPreviouslyRemovedConnectors(TomcatWebServer.java:263)
	at org.springframework.boot.web.embedded.tomcat.TomcatWebServer.start(TomcatWebServer.java:195)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.startWebServer(ServletWebServerApplicationContext.java:297)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.finishRefresh(ServletWebServerApplicationContext.java:163)
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:552)
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:141)
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:743)
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:390)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:312)
	at org.springframework.boot.builder.SpringApplicationBuilder.run(SpringApplicationBuilder.java:140)
	at cc.mrbird.febs.FebsApplication.main(FebsApplication.java:17)
Caused by: java.net.BindException: Address already in use: bind
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:444)
	at sun.nio.ch.Net.bind(Net.java:436)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:225)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:74)
	at org.apache.tomcat.util.net.NioEndpoint.initServerSocket(NioEndpoint.java:230)
	at org.apache.tomcat.util.net.NioEndpoint.bind(NioEndpoint.java:213)
	at org.apache.tomcat.util.net.AbstractEndpoint.bindWithCleanup(AbstractEndpoint.java:1124)
	at org.apache.tomcat.util.net.AbstractEndpoint.start(AbstractEndpoint.java:1210)
	at org.apache.coyote.AbstractProtocol.start(AbstractProtocol.java:585)
	at org.apache.catalina.connector.Connector.startInternal(Connector.java:1005)
	... 13 common frames omitted
2020-12-24 21:06:36.370 febs [main] ERROR o.s.b.d.LoggingFailureAnalysisReporter - 

***************************
APPLICATION FAILED TO START
***************************

Description:

The Tomcat connector configured to listen on port 9527 failed to start. The port may already be in use or the connector may be misconfigured.

Action:

Verify the connector's configuration, identify and stop any process that's listening on port 9527, or configure this application to listen on another port.

2020-12-24 21:14:29.755 febs [http-nio-9527-exec-6] ERROR c.m.f.c.authentication.JWTFilter - token已经过期
2020-12-24 21:14:29.814 febs [http-nio-9527-exec-6] ERROR c.m.f.c.authentication.JWTFilter - token已经过期
